src/status_im/ui/screens/chat/message/link_preview.cljs

```diff
@@ -75,13 +78,14 @@
             [react/image {:source              {:uri thumbnailUrl}
                           :style               (styles/link-preview-image outgoing)
                           :accessibility-label :member-photo}]
-            [quo/text {:size  :small
-                       :style styles/link-preview-title}
-             title]
-            [quo/text {:size  :small
-                       :color :secondary
-                       :style styles/link-preview-site}
-             site]]])))))
+            (when-not (is-gif? thumbnailUrl)
+              [quo/text {:size  :small
```

Comment:
you need to add `[:<>` then, because only the last form will be returned

Reply:
Done